<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIMemoryTransfer</class>
 <widget class="QDialog" name="UIMemoryTransfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory Transfer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Start Address:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leStartAddress"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="leEndAddress"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>End Address:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>File:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leFile"/>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="tbBrowse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="rbDownload">
         <property name="text">
          <string>Download</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbUpload">
         <property name="text">
          <string>Upload</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cbPC">
       <property name="text">
        <string>Set PC to Start Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dbbButtons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dbbButtons</sender>
   <signal>accepted()</signal>
   <receiver>UIMemoryTransfer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbButtons</sender>
   <signal>rejected()</signal>
   <receiver>UIMemoryTransfer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
